js/web/karma.conf.js

```diff
@@ -90,6 +90,8 @@ module.exports = function(config) {
     hostname,
     listenAddress,
     customLaunchers: {
+      // the following flags are used to make sure Edge on CI agents to initialize WebGPU correctly.
+      EdgeWebGpuTest: {base: 'Edge', flags: ['--ignore-gpu-blocklist', '--gpu-vendor-id=0x10de']},
```

Comment:
this is the nvidia id - would be good to add intel's id

Reply:
We use this parameter to specific with GPU to run. So, if your machine has an Intel GPU and a Nvidia GPU, we use this parameter to tell the test runner to use the Nvidia one.  Otherwise chrome may choose one that totally doesn't support webgpu(at the moment when chrome is launching, it doesn't know you are going to open a webpage that requires webgpu)

Reply:
probably be a good idea to make this flag passing from yml.